raster/r.univar/testsuite/benchmark_r_univar.py

```diff
@@ -0,0 +1,59 @@
+"""Benchmarking of r.univar
```

Comment:
test_r_univar.py should not be deleted. This benchmark_r_univar.py file is extra.

Reply:
Fails on my Linux machine too with the same message. What is the min version of GCC that this parallel code supports?

Reply:
* https://stackoverflow.com/a/13770822/16079666
* https://stackoverflow.com/a/13203075/16079666

Reply:
FYI, gcc 5.5.0

Reply:
OK, all those variables are `const` and removing them from `shared` worked. I think `const`s are shared by default.

Reply:
I think I must've accidentally deleted the file when separating the scripts to respective directories, will fix it.